groups.adoc

```diff
@@ -0,0 +1,53 @@
+[#groups]
+= Import groups from the IDP provider for OCP 4.10+
+
+
+[#introduction]
+== Introduction
+
+In OpenShift 4.10, a new capability was added to import groups from an IDP provder. For example, while using LDAP, you can import the groups of which a user is member of. This allows you to create RBAC based on your IDP provider group.
+
+[#LDAP Configuration]
+== LDAP Configuration
+
+In the Authrealm, you can add a section explaining how to query the groups the user belong too.
+
+Let's say for example you have this ldap entries.
+
+[source,shell]
+----
+# testuser, example.com
+dn: cn=testuser,dc=example,dc=com
+objectClass: person
+objectClass: inetOrgPerson
+sn: user
+cn: test
+cn: testuser
+mail: testuser@example.com
+userPassword:: Zm9vYmFy
+
+# mygroup, example.com
+dn: cn=mygroup,dc=example,dc=com
+objectClass: groupOfNames
+cn: mygroup
+member: cn=testuser,dc=example,dc=com
+----
```

Comment:
In this example, there doesn't seem to be a relationship between `testuser@example.com` and `mygroup`. Is that expected or are we missing `objectClass:groupOfNames` in the first entry?

Reply:
the relationship is in `member: cn=testuser,dc=example,dc=com` which declare testuser part of mygroup

Reply:
oops, I missed that!